ProblemZ.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    long double A, B, C, D;
+    cin >> A >> B >> C >> D;
+
+    long double new_B = B/10000000000000;
```

Comment:
(1) You can add a comment why you chose this number.

(2) You may want to use digit separator. You'll find something similar in almost every language: https://www.geeksforgeeks.org/digit-separator-in-c14/